src/theme/SearchBar/searchbar.css

```diff
@@ -0,0 +1,91 @@
+.searchbox {
```

Comment:
Let's rename this file to `styles.css` to it aligns a bit closer to other components (even though they're able to use modules)

Reply:
And then in this file if you could just add comments above the selectors noting what that specific styling does then that will help a future contributor a lot